src/Management/src/Endpoint/Actuators/CloudFoundry/CloudFoundrySecurityMiddleware.cs

```diff
@@ -156,7 +169,7 @@ internal Task<SecurityResult> GetPermissionsAsync(HttpContext context)
 
     private async Task ReturnErrorAsync(HttpContext context, SecurityResult error)
     {
-        LogError(context, error);
+        _logger.LogError("Actuator Security Error: {Code} - {Message}", error.Code, error.Message);
```

Comment:
Would it be helpful to at least list the header names?

Reply:
I think the logging of headers was added before there was a robust middleware available from Microsoft to handle it... I think I'd rather suggest using that to diagnose problems than bundling any form of it here

Reply:
The Microsoft middleware handles ASP.NET header logging, but this is about an external call using `HttpClient`.

Reply:
See also https://learn.microsoft.com/en-us/dotnet/core/compatibility/networking/9.0/redact-headers.

Reply:
The built-in trace logging is there not matter what we do, so Steeltoe logging was definitely excessive. I've added a call to `RedactLoggedHeaders` and specified Authorization so that the token is not logged